qemu/tests/cfg/qemu_disk_img_create.cfg

```diff
@@ -0,0 +1,57 @@
+- qemu_disk_img_create:
+    no qed raw vmdk
+    type = create_external_snapshot_via_protocol
+    kill_vm = yes
+    start_vm = no
+    md5sum_bin = "md5sum"
+    force_create_image = no
+    backup_image_before_testing = yes
+    restore_image_before_testing = yes
```

Comment:
I don't thinks so. as we create a template file in the base image. If using snapshot, we could not find it in the snapshot image.